Sources/TuistDependencies/Carthage/CarthageInteractor.swift

```diff
@@ -91,40 +91,33 @@ public final class CarthageInteractor: CarthageInteracting {
             throw CarthageInteractorError.carthageNotFound
         }
 
-        // install dependencies and generate dependencies graph
-        let dependenciesGraph: TuistCore.DependenciesGraph = try FileHandler.shared
-            .inTemporaryDirectory { temporaryDirectoryPath in
-                // prepare paths
-                let pathsProvider = CarthagePathsProvider(
-                    dependenciesDirectory: dependenciesDirectory,
-                    temporaryDirectoryPath: temporaryDirectoryPath
-                )
-
-                // prepare for installation
-                try loadDependencies(pathsProvider: pathsProvider, dependencies: dependencies)
-
-                // run `Carthage`
-                if shouldUpdate {
-                    try carthageController.update(
-                        at: temporaryDirectoryPath,
-                        platforms: platforms,
-                        printOutput: true
-                    )
-                } else {
-                    try carthageController.bootstrap(
-                        at: temporaryDirectoryPath,
-                        platforms: platforms,
-                        printOutput: true
-                    )
-                }
-
-                // post installation
-                try saveDependencies(pathsProvider: pathsProvider)
-
-                // generate dependencies graph
-                return try carthageGraphGenerator
-                    .generate(at: pathsProvider.temporaryCarthageBuildDirectory)
-            }
+        // prepare paths
```

Comment:
`Prepare paths` is a bit vague and it's hard to know for what if you don't have context in this area of the codebase. What are those paths? Why do they need to be prepared?